packages/vscode-apollo/src/testRunner/jest-vscode-environment.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Exposes the Visual Studio Code extension API to the Jest testing environment.
+ * For custom environments reference:
+ * @see https://jestjs.io/docs/en/configuration.html#testenvironment-string
+ */
+
+// Using import here results in `jest_environment_node_1.default`
+// The `.default` in this case doesn't exist. Interop issue with the built
+// version of the package?
+const NodeEnvironment = require("jest-environment-node");
+import * as vscode from "vscode";
+
+class VsCodeEnvironment extends NodeEnvironment {
+  constructor(config: any) {
```

Comment:
we shouldn't need the constructor + super since the super is the only action and is called with the config